converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java

```diff
@@ -101,7 +100,7 @@ private JsonWrapper encodeMeasurementSets(Map<TemplateId, Node> childMapByTempla
 					String message = "No encoder for decoder : " + child.getType();
 					throw new EncodeException(message, exc);
 				}
-			}
+//			}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)